src/nlp_types.jl

```diff
@@ -168,36 +168,57 @@ end
 
 # Displaying NLPModelMeta instances.
 
-import Base.show, Base.print
+import Base.show, Base.print, Base.println
 function show(io :: IO, nlp :: NLPModelMeta)
   s  = nlp.minimize ? @sprintf("Minimization ") : @sprintf("Maximization ")
   s *= @sprintf("problem %s\n", nlp.name)
   s *= @sprintf("nvar = %d, ncon = %d (%d linear)\n", nlp.nvar, nlp.ncon, nlp.nlin)
   print(io, s)
 end
 
-function print(io :: IO, nlp :: NLPModelMeta)
-  nlp.minimize ? @printf("Minimization ") : @printf("Maximization ")
-  @printf("problem %s\n", nlp.name)
-  @printf("nvar = %d, ncon = %d (%d linear)\n", nlp.nvar, nlp.ncon, nlp.nlin)
-  @printf("lvar = "); display(nlp.lvar'); @printf("\n")
-  @printf("uvar = "); display(nlp.uvar'); @printf("\n")
-  @printf("lcon = "); display(nlp.lcon'); @printf("\n")
-  @printf("ucon = "); display(nlp.ucon'); @printf("\n")
-  @printf("x0 = ");   display(nlp.x0'); @printf("\n")
-  @printf("y0 = ");   display(nlp.y0'); @printf("\n")
-  @printf("nnzh = %d\n", nlp.nnzh);
-  @printf("nnzj = %d\n", nlp.nnzj);
+"""
+    print(io, meta; verbose=false)
+
+Prints meta information - x0, nvar, ncon, etc.
+If `verbose` is `false`, then vectors with more than 5 componens show only the
+first 4 and the last, like
+
+    1.0 2.0 3.0 4.0 ... 10.0
+"""
+function print(io :: IO, nlp :: NLPModelMeta; verbose=false)
+  nlp.minimize ? @printf(io, "Minimization ") : @printf(io, "Maximization ")
```

Comment:
I think we could get rid of `verbose`, couldn't we? If a user doesn't want details, they should call `show()`.